OpenRA.Mods.Common/Projectiles/LaserZap.cs

```diff
@@ -23,6 +23,9 @@ namespace OpenRA.Mods.Common.Projectiles
 	[Desc("Not a sprite, but an engine effect.")]
 	public class LaserZapInfo : IProjectileInfo
 	{
+		[Desc("Should the beam be visually rendered? False = Beam is invisible.")]
+		public readonly bool RenderBeam = true;
```

Comment:
A laser that isn't rendered isn't a laser at all (just a normal projectile), so isn't suited for this projectile type at all.